tests/Model/Document/DocumentTest.php

```diff
@@ -367,16 +367,4 @@ public function testDocumentSerialization(): void
 
         $this->assertEquals(count($document->getEditables()), count($loadedDocument->getEditables()));
     }
-
-    public function testDocumentPrint(): void
```

Comment:
This is the only test connected to the web2print bundle and we decided to remove it so we do not have a dependency in the Core. 
We also decided that we do not move this one test to the bundle since the overhead for just one test would be too much. 
I will remove the uses.